include/SSVOpenHexagon/Core/HexagonGame.hpp

```diff
@@ -264,12 +264,20 @@ class HexagonGame
 
     void printLuaDocs()
     {
-        luaMetadata.forFnEntries(
-            [](const std::string& ret, const std::string& name,
-                const std::string& args, const std::string& docs) {
-                std::cout << "* **`" << ret << " " << name << "(" << args
-                          << ")`**: " << docs << '\n';
-            });
+        for (std::size_t i = 0; i < luaMetadata.getNumCategories(); ++i) 
+        {
+            std::cout << "\n" 
+                      << luaMetadata.prefixHeaders.at(i) 
+                      << "\n" << std::endl;
```

Comment:
```suggestion
            std::cout << '\n' 
                      << luaMetadata.prefixHeaders.at(i) 
                      << "\n\n";
```

Very minor stylistic suggestion - we don't need `endl` here

Reply:
I did it because when I did 2 "\n"s, the compiler threw warnings about it. But sure this works as well